.gitignore

```diff
@@ -10,3 +10,4 @@ ipa-web.env
 out/
 *.log
 bin/
+.DS_Store
```

Comment:
You should not be updating .gitignore in this PR. If you want to update the .gitignore, please make a PR which does exclusively that.